README.md

```diff
@@ -1,24 +1,71 @@
+1. ~~Go to https://www.anaconda.com/distribution/~~
+2. ~~Download Anaconda for Python 3~~
+3. ~~Complete the installation. During installation, make sure the following options are checked:~~
+    - [x] ~~Add Anaconda to my PATH environment variable~~
+    - [x] ~~Register Anaconda as my default Python~~
+    - [x] ~~Install Anaconda for local user, rather than all users~~
 
-2. Unzip the download folder and run the python script install-miniforge.py
+## Steps to Uninstall an NHRAP-HAZUS Open Source tool
+1. Delete the tool's unzipped folder and contents
+
+## Steps to uninstall Anaconda
+Older versions of the NHRAP HAZUS open source tools use Anaconda to install and manager the Python environment(s). It is recommended to uninstall the older versions and Anaconda and use the latest tool versions with Miniforge.
+
+1. Using Anaconda, delete any virtual environments
+    * This can be done via the Anaconda program interface or via [terminal window or Anaconda Prompt](https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html#removing-an-environment).
+    * Via Anaconda Program Interface:
+      1. Open Anaconda.
+      2. Select the 'hazus_env' or 'hazus_\<tool acronym\>_env'.
+      3. Click the 'Remove' button.
+      4. A popup will appear asking you to confirm. Click the 'Remove' button. This may take a few minutes.
+      5. Repeat steps 2 - 3 until all 'hazus_...' virtual environments are removed.
+      ![Remove ENV Anaconda](images/AnacondaRemoveEnv.jpg "Remove ENV Anaconda")
+    * Via terminal window:
+      1. Open a command line prompt
+      2. Enter the following without qoutes 'conda info --envs' to see your environments. 'hazus_env' should be listed.
+      3. Enter the following without quotes 'conda env remove --name hazus_env' to remove the environment. This may take a few minutes.
+      4. Enter the following without qoutes 'conda info --envs' to see your environments again and hazus_env should not be listed.
+      5. Repeat steps 2 - 3 until all 'hazus_...' virtual environments are removed.
+      ![Remove ENV Terminal](images/CommandLineRemoveEnv.jpg "Remove ENV Terminal")
+
+2. Follow the [uninstall steps](https://docs.anaconda.com/anaconda/install/uninstall/) from Anaconda 
+    * It is recommended to perform the 'Option B. Full uninstall using Anaconda-Clean and simple remove.'
+
+## Environment and Tool Troubleshooting
+
+**Tool is not opening with Python.**
+To ensure .py files run when double-clicked, right-click the .py file and go to Properties. Under the "General" tab next to "Opens With", make sure "python.exe" is selected. If not, click "Change" and select "python.exe" from your Python installation directory.
 
 
 ## Contact
 
-Issues can be reported through the repository on Github (https://github.com/nhrap-hazus/Environment-Setup)
+Issues can be reported through this [repository](https://github.com/nhrap-hazus/Environment-Setup) on Github.
```

Comment:
Change wording of this to active tone & capitalize the 'H' in GitHub. Something like the following:

_"Please report issues through the Environment Setup GitHub repository: https://github.com/nhrap-hazus/Environment-Setup"_